src/you_get/extractors/mgtv.py

```diff
@@ -75,8 +75,9 @@ def prepare(self, **kwargs):
                 size_this_stream = 0
                 stream_fileid_list = []
                 for i in segment_list_this:
-                    _, container_this_stream, size_this_seg = url_info(i)
-                    size_this_stream += size_this_seg
+                    if 'json_output' not in kwargs and not kwargs['json_output']:
```

Comment:
I don't think so.

As I can remember MGTV's video URL is only **one** for each resolution(total resolution is 3 for most of the cases), backed by ChinaNet(WS): If the ```url_info``` fails, I can hardly imagine that the video can be downloaded by any chance.